language/stdlib/modules/libra_account.move

```diff
@@ -20,14 +25,15 @@ module LibraAccount {
         // If true, the authority to withdraw funds from this account resides elsewhere
         delegated_withdrawal_capability: bool,
         // Event handle for received event
-        received_events: EventHandle<ReceivedPaymentEvent>,
+        received_events: Event::EventHandle<ReceivedPaymentEvent>,
         // Event handle for sent event
-        sent_events: EventHandle<SentPaymentEvent>,
+        sent_events: Event::EventHandle<SentPaymentEvent>,
         // The current sequence number.
         // Incremented by one each time a transaction is submitted
         sequence_number: u64,
-        // Generator for event handles
-        event_generator: EventHandleGenerator,
+        is_frozen: bool,
+        // The ticker string for the balance held by this account.
+        balance_currency_ticker: vector<u8>,
```

Comment:
We might eventually want a wrapper type for this if it is used in so many places.

Reply:
Agreed, my only thinking is that this should be treated completely opaquely by Move -- only the VM/clients actually interpret this "blob" as anything meaningful. So I'm somewhat hesitant to add a newtype just for this.

That being said, I do think we might want to introduce a String module that holds the encoding info as well. If we do that then I think using that here would make sense.

Reply:
Good points re: uninterpretedness in Move